ur_description/urdf/ur.ros2_control.xacro

```diff
@@ -56,8 +64,9 @@
         <state_interface name="position"/>
         <state_interface name="velocity"/>
         <state_interface name="effort"/>
+        <param name="initial_position">-1.57</param>  <!-- initial position for the FakeSystem -->
```

Comment:
Do you want to define an initial position for the other joints or is the default OK?

Reply:
I think it's great to have an example of setting this for one joint  :+1: